src/components/link/link.js

```diff
@@ -165,9 +165,9 @@ export const BLink = /*#__PURE__*/ Vue.extend({
         target: this.target,
         tabindex: this.disabled
           ? '-1'
-          : isUndefined(this.$attrs.tabindex)
-            ? null
-            : this.$attrs.tabindex,
+          : this.$attrs && this.$attrs.tabindex
```

Comment:
This will fail if `tabindex === 0` (passed as a number)

A better solution would be to set a temp variable `const $attrs = this.$attrs || {}`

And then return the test to:

```js
        tabindex: this.disabled
          ? '-1'
          : isUndefined($attrs.tabindex)
            ? null
            : $attrs.tabindex,
```

Although `this.$attrs` is supposed to always available (as an object) in the latest Vue.